src/main/java/hudson/plugins/clover/CloverProjectAction.java

```diff
@@ -26,9 +26,9 @@
 
     static final String ICON = "/plugin/clover/clover_48x48.png";
     
-    private final Project<?, ?> project;
+    private transient final Job<?, ?> project;
```

Comment:
I think it is not needed to serialize this information.
However - I'm not an expert and this may be just a "embedded disease" - do not use to much resources.
Nevertheless I think that ProjectAction is not serialized at all - it is each time created on-the-fly by onLoad method from BuildAction.